### ISSUE TITLE
Active filter labels

### ISSUE THREAD

**Issue by artlowel** [DSpace/dspace-angular#173]
Create a component to show the active filters as labels on the search page, underneath the search box [as shown in the mockups on the wiki](https://wiki.duraspace.org/display/DSPACE/Search+Mockup#SearchMockup-Selectingfilters)

Add a method to the SearchService that will return the list of active filters based on the URL. The `paramName`s you get back from `SearchService.getConfig()` (e.g. `f.author`, `f.scope`, `f.date`, …) are the request params in the url.

e.g. it the url is `https://dspace7-demo.atmire.com/search?query=influenza%20hospital&f.author=Doe%2C%20John&f.author=Smith%2C%20Donald%20Jr.&f.scope=94c6458d-277c-4b04-b328-4e88a5ceec76&view=grid&f.date=2009` this new method should return 
``` typescript
[
    {
        value: 'Doe, John',
        removeLink: 'https://dspace7-demo.atmire.com/search?query=influenza%20hospital&f.author=Smith%2C%20Donald%20Jr.&f.scope=94c6458d-277c-4b04-b328-4e88a5ceec76&view=grid&f.date=2009'
    },
    {
        value: 'Smith, Donald Jr.',
        removeLink: 'https://dspace7-demo.atmire.com/search?query=influenza%20hospital&f.author=Doe%2C%20John&f.scope=94c6458d-277c-4b04-b328-4e88a5ceec76&view=grid&f.date=2009'
    },
    {
        value: '94c6458d-277c-4b04-b328-4e88a5ceec76',
        removeLink: 'https://dspace7-demo.atmire.com/search?query=influenza%20hospital&f.author=Doe%2C%20John&f.author=Smith%2C%20Donald%20Jr.&view=grid&f.date=2009'
    },
    {
        value: '2009',
        removeLink: 'https://dspace7-demo.atmire.com/search?query=influenza%20hospital&f.author=Doe%2C%20John&f.author=Smith%2C%20Donald%20Jr.&f.scope=94c6458d-277c-4b04-b328-4e88a5ceec76&view=grid'
    },
]
```

Use the data from this new SearchService method to render the filter labels.

Each label should have a "close"-link (the `x`) with the `removeLink` property as their href

**Comment by artlowel** [comment]
closed by #199

**Comment by artlowel** [comment]
Reopening: closed by mistake
